lab/runnerctl.py

```diff
@@ -32,13 +31,23 @@ def pytest_namespace():
     return {'halt': Halt()}
 
 
+def pytest_configure(config):
+    signal.signal(signal.SIGTERM, exit_gracefully)
```

Comment:
Cross that bridge when someone complains? We're neither violating the semantics of that signal, nor should we be changing behaviour people would be relying on.